install-monorepo.sh

```diff
@@ -58,11 +62,11 @@ fi
 
 echo "Checking if NVM is installed..."
 if ! command -v nvm &>/dev/null; then
-    echo "Installing nvm"
-    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash && export NVM_DIR=$HOME/.nvm && source $NVM_DIR/nvm.sh
+	echo "Installing nvm"
+	curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash && export NVM_DIR=$HOME/.nvm && source $NVM_DIR/nvm.sh  --no-use
 else
 	echo "Updating nvm"
-    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+	curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash  || true
```

Comment:
Is the `|| true` needed here? Or was it the lack of `--no-use` on line 66 that was causing the failure before?

Reply:
Yup, this seems to be the case. Resolved.